config-templates/redhat/odf-local/changelog.json

```diff
@@ -1,29 +1,46 @@
 [
     {
-        "4.16": {
+        "4.17": {
             "status": "supported",
             "latest-revision": {
-                "revision": "2",
-                "release-date": "13 December 2024",
+                "revision": "1",
+                "release-date": "22 January 2024",
                 "cves": "",
-                "golang": "1.22.7",
+                "golang": "1.22.9",
                 "ubi": "",
                 "new-features": [
                 ],
                 "fixes": [
-                    "Worker Pool Parameter Bug fix"
+                    "Initial release 4.17"
+                ]
+            }
+        }
+    },
+    {
+        "4.16": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "4",
+                "release-date": "22 January 2025",
+                "cves": "CVE-2023-37920,CVE-2024-3596,CVE-2024-5535",
+                "golang": "1.22.9",
+                "ubi": "",
+                "new-features": [
+                ],
+                "fixes": [
+                    "VA fix"
                 ]
             },
             "previous-revision": {
-                "revision": "1",
-                "release-date": "12 October 2024",
+                "revision": "3",
```

Comment:
when revision 3 was released? I think it was never released. Please confirm and update accordingly